README.md

```diff
@@ -5,45 +5,44 @@
   <h6>Orchestrate like never before. K8s reimagined.</h6>
   <p align="center">
     <a href="https://next-hat.com/nanocl"><b>Why</b></a> •
-    <a href="https://docs.next-hat.com/manuals/nanocl/install/overview"><b>Install</b></a> •
-    <a href="https://discord.gg/WV4Aac8uZg" target="_blank"><b>Discord</b></a> •
+    <a href="https://docs.next-hat.com/manuals/nanocl/install/overview"><b>Quickstart</b></a> •
```

Comment:
i think the quick start should more point out to the getting started overview guide